examples/subplots_axes_and_figures/mosaic.py

```diff
@@ -1,14 +1,10 @@
 """
+.. redirect-from:: /tutorials/provisional/mosaic
+
 =======================================
 Complex and semantic figure composition
```

Comment:
```suggestion
subplot_mosaic: Complex and semantic figure composition
```

Reply:
Good point!  I did it silghtly differently and adjusted the decoration so just push a new commit.